charts/argo-cd/templates/crds/crd-applicationset.yaml

```diff
@@ -897,13 +1464,13 @@ spec:
                               required:
                               - destination
                               - project
-                              - source
                               type: object
                           required:
                           - metadata
                           - spec
                           type: object
                       required:
+                      - pathParamPrefix
```

Comment:
Is it intentional that this is required now? I'm getting the following error when upgrading:

```
failed to apply ApplicationSet.argoproj.io, argocd/skip-apps: ApplicationSet.argoproj.io "skip-apps" is invalid: spec.generators[0].git.         pathParamPrefix: Required value
```

The documentation seems to infer that it should be optional: 
https://argo-cd.readthedocs.io/en/release-2.6/operator-manual/applicationset/Generators-Git/#git-generator-directories

Reply:
Good catch @esphen - missed that one with diff. PR contains the fix https://github.com/argoproj/argo-helm/pull/1829

Reply:
ASAIK it was an upstream error, but solved in v2.6.0-rc3 here: https://github.com/argoproj/argo-cd/pull/11943

So probably these CRDs are not up-to-date?